.pre-commit-config.yaml

```diff
@@ -10,3 +10,18 @@ repos:
       - id: isort
         name: isort (python)
         args: [--profile, black]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: 'v8.4.0'
+    hooks:
+    -   id: eslint
+        files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
+        types: [file]
+        args: [--resolve-plugins-relative-to, ./pyscriptjs]
+        additional_dependencies:
+        - 'eslint'
+        - 'eslint-plugin-svelte3'
+        - 'typescript'
+        - '@typescript-eslint/eslint-plugin'
+        - '@typescript-eslint/parser'
```

Comment:
Are these @ really supposed to be there?

Reply:
As far as I can tell, yes. The package names are like that in the package.json file. I'd you remove them you'll get an installation error. 

Essentially, we need to create a minimal node environment for running the linter in an isolated environment.

Reply:
TIL it has to do with namespace scopes.

https://stackoverflow.com/a/49152240